cropro.py

```diff
@@ -49,11 +50,13 @@ def __init__(self, *args, **kwargs):
         self.search_result_label = SearchResultLabel()
         self.into_profile_label = ProfileNameLabel()
         self.current_profile_deck_combo = DeckCombo()
+        self.edit_button = QPushButton('Edit')
         self.import_button = QPushButton('Import')
         self.search_term_edit = QLineEdit()
         self.other_profile_names_combo = ComboBox()
         self.other_profile_deck_combo = DeckCombo()
         self.filter_button = QPushButton('Filter')
+        self.help_button = QPushButton('Help')
```

Comment:
Not sure why this button is needed here.

Reply:
I myself didn't know that this was a standard anki search field and didn't know about the possibilities of it so I thought it would be useful